server/ngwmn/templates/macros/components.html

```diff
@@ -1,4 +1,13 @@
-{% macro Graph(agency_code, site_no) -%}
+{# This component will request a remote resource and then insert it as a child into the DOM #}
+{% macro SVGPattern(url, id) %}
+    <div class="ngwmn-component" data-component="svg-pattern" data-url="{{ url }}" data-id="{{ id }}"></div>
```

Comment:
This component inserts an external SVG document into the DOM in a form usable as a fill pattern.